lib/Widget/shared/alert_style.dart

```diff
@@ -81,4 +81,26 @@ class Dialogs {
       },
     );
   }
+
+  static Future<void> alertDashboardDetail(BuildContext context) async {
+    await showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+            shape:
+                RoundedRectangleBorder(borderRadius: BorderRadius.circular(15)),
+            title: Text("คนไข้ยังไม่มีข้อมูลในขั้นตอนการรักษานี้",
```

Comment:
Consider change 'คนไข้' to 'ผู้ป่วย'.
Because we mostly use that word.

Reply:
Fixed in dca9edc1a227d360bb883bf47a298989d054ea3a